maintainers/maintainer-list.nix

```diff
@@ -15848,574 +16417,5 @@
-    githubId = 102890144;
-  };
-  macalinao = {
-    email = "me@ianm.com";
-    name = "Ian Macalinao";
-    github = "macalinao";
-    githubId = 401263;
-    keys = [{
-      fingerprint = "1147 43F1 E707 6F3E 6F4B  2C96 B9A8 B592 F126 F8E8";
-    }];
-  };
-  tjni = {
-    email = "43ngvg@masqt.com";
-    matrix = "@tni:matrix.org";
-    name = "Theodore Ni";
-    github = "tjni";
-    githubId = 3806110;
-    keys = [{
-      fingerprint = "4384 B8E1 299F C028 1641  7B8F EC30 EFBE FA7E 84A4";
-    }];
-  };
-  bezmuth = {
-    email = "benkel97@protonmail.com";
-    name = "Ben Kelly";
-    github = "bezmuth";
-    githubId = 31394095;
-  };
-  cafkafk = {
-    email = "christina@cafkafk.com";
-    matrix = "@cafkafk:matrix.cafkafk.com";
-    name = "Christina Sørensen";
-    github = "cafkafk";
-    githubId = 89321978;
-    keys = [
-      {
-        fingerprint = "7B9E E848 D074 AE03 7A0C  651A 8ED4 DEF7 375A 30C8";
-      }
-      {
-        fingerprint = "208A 2A66 8A2F CDE7 B5D3 8F64 CDDC 792F 6552 51ED";
-      }
-    ];
-  };
-  rb = {
-    email = "maintainers@cloudposse.com";
-    github = "nitrocode";
-    githubId = 7775707;
-    name = "RB";
-  };
-  bpaulin = {
-    email = "brunopaulin@bpaulin.net";
-    github = "bpaulin";
-    githubId = 115711;
-    name = "bpaulin";
-  };
-  zuzuleinen = {
-    email = "andrey.boar@gmail.com";
-    name = "Andrei Boar";
-    github = "zuzuleinen";
-    githubId = 944919;
-  };
-  waelwindows = {
-    email = "waelwindows9922@gmail.com";
-    github = "Waelwindows";
-    githubId = 5228243;
-    name = "waelwindows";
-  };
-  witchof0x20 = {
-    name = "Jade";
-    email = "jade@witchof.space";
-    github = "witchof0x20";
-    githubId = 36118348;
-    keys = [{
-      fingerprint = "69C9 876B 5797 1B2E 11C5  7C39 80A1 F76F C9F9 54AE";
-    }];
-  };
-  WhiteBlackGoose = {
-    email = "wbg@angouri.org";
-    github = "WhiteBlackGoose";
-    githubId = 31178401;
-    name = "WhiteBlackGoose";
-    keys = [{
-      fingerprint = "640B EDDE 9734 310A BFA3  B257 52ED AE6A 3995 AFAB";
-    }];
-  };
-  wuyoli = {
-    name = "wuyoli";
-    email = "wuyoli@tilde.team";
-    github = "wuyoli";
-    githubId = 104238274;
-  };
-  ziguana = {
-    name = "Zig Uana";
-    email = "git@ziguana.dev";
-    github = "ziguana";
-    githubId = 45833444;
-  };
-  detegr = {
-    name = "Antti Keränen";
-    email = "detegr@rbx.email";
-    github = "Detegr";
-    githubId = 724433;
-  };
-  RossComputerGuy = {
-    name = "Tristan Ross";
-    email = "tristan.ross@midstall.com";
-    github = "RossComputerGuy";
-    githubId = 19699320;
-  };
-  franzmondlichtmann = {
-    name = "Franz Schroepf";
-    email = "franz-schroepf@t-online.de";
-    github = "franzmondlichtmann";
-    githubId = 105480088;
-  };
-  srounce = {
-    name = "Samuel Rounce";
-    email = "me@samuelrounce.co.uk";
-    github = "srounce";
-    githubId = 60792;
-  };
 }
+/* Keep the list alphabetically sorted. */
```

Comment:
Are we losing a line somewhere? Both files have 16421 lines, while this one has an extra comment.

Reply:
There was a blank line deleted between two entries.